"use strict";(self.webpackChunkgenomics_docusaurus_new=self.webpackChunkgenomics_docusaurus_new||[]).push([[753],{3282:(e,i,n)=>{n.d(i,{A:()=>s});const s=n.p+"assets/images/kinase-integration-579226a7109c5547f69695884fb56810.png"},8453:(e,i,n)=>{n.d(i,{R:()=>o,x:()=>a});var s=n(6540);const t={},r=s.createContext(t);function o(e){const i=s.useContext(r);return s.useMemo((function(){return"function"==typeof e?e(i):{...i,...e}}),[i,e])}function a(e){let i;return i=e.disableParentContext?"function"==typeof e.components?e.components(t):e.components||t:o(e.components),s.createElement(r.Provider,{value:i},e.children)}},9580:(e,i,n)=>{n.r(i),n.d(i,{assets:()=>l,contentTitle:()=>a,default:()=>p,frontMatter:()=>o,metadata:()=>s,toc:()=>d});const s=JSON.parse('{"id":"graphdb/index","title":"Overview","description":"Integrating Kinase Bioactivity with Expression Profiles","source":"@site/docs/graphdb/index.md","sourceDirName":"graphdb","slug":"/graphdb/","permalink":"/datasolutions/graphdb/","draft":false,"unlisted":false,"tags":[],"version":"current","sidebarPosition":1,"frontMatter":{"id":"index","title":"Overview","sidebar_position":1,"hide_title":true},"sidebar":"docsSidebar","previous":{"title":"Overview","permalink":"/datasolutions/mimic-onco/"},"next":{"title":"Examples","permalink":"/datasolutions/graphdb/examples"}}');var t=n(4848),r=n(8453);const o={id:"index",title:"Overview",sidebar_position:1,hide_title:!0},a=void 0,l={},d=[{value:"Integrating Kinase Bioactivity with Expression Profiles",id:"integrating-kinase-bioactivity-with-expression-profiles",level:2},{value:"Pipeline Overview",id:"pipeline-overview",level:2},{value:"Use Cases",id:"use-cases",level:2}];function c(e){const i={em:"em",h2:"h2",hr:"hr",li:"li",p:"p",strong:"strong",ul:"ul",...(0,r.R)(),...e.components};return(0,t.jsxs)(t.Fragment,{children:[(0,t.jsx)(i.h2,{id:"integrating-kinase-bioactivity-with-expression-profiles",children:"Integrating Kinase Bioactivity with Expression Profiles"}),"\n",(0,t.jsx)(i.hr,{}),"\n",(0,t.jsx)("div",{style:{textAlign:"justify"},children:(0,t.jsxs)(i.p,{children:["This project involves integration of reference data and modeling using semantic web technologies. As a proof of concept, it combines ",(0,t.jsx)(i.strong,{children:"kinase-targeted bioactivity data (ChEMBL)"})," with ",(0,t.jsx)(i.strong,{children:"normal (GTEx)"})," and ",(0,t.jsx)(i.strong,{children:"tumor (TCGA)"})," gene expression profiles, harmonized via ",(0,t.jsx)(i.strong,{children:"UniProt ID mappings"})," and exposed through a ",(0,t.jsx)(i.strong,{children:"GraphDB triple store"}),"."]})}),"\n",(0,t.jsx)(i.h2,{id:"pipeline-overview",children:"Pipeline Overview"}),"\n",(0,t.jsx)(i.hr,{}),"\n",(0,t.jsx)("div",{style:{textAlign:"center"},children:(0,t.jsx)("img",{src:n(3282).A,alt:"Integration Pipeline",style:{width:"130%",maxWidth:"1300px",marginLeft:"-10%"}})}),"\n",(0,t.jsxs)("div",{style:{textAlign:"justify"},children:[(0,t.jsx)("br",{}),(0,t.jsx)("br",{}),(0,t.jsx)(i.p,{children:"Diagram illustrating the modular RDF-based data integration pipeline. Each dataset (ChEMBL, GTEx, TCGA, and UniProt) is processed independently and exported to Turtle (TTL) format, then loaded into GraphDB as named graphs. These datasets are harmonized through ontology-aligned identifiers (e.g., UniProt \u2194 Ensembl \u2194 UBERON) to enable semantic joins across:"}),(0,t.jsx)("div",{style:{textAlign:"center",fontWeight:"bold"},children:(0,t.jsx)(i.p,{children:"Compound \u2192 Kinase \u2192 Gene \u2192 Expression"})}),(0,t.jsx)("br",{}),(0,t.jsxs)(i.p,{children:["The pipeline supports cross-dataset querying, allowing questions such as: ",(0,t.jsx)(i.em,{children:'"Which compounds target kinases that are overexpressed in kidney tumors but not in healthy tissue?"'})]}),(0,t.jsxs)(i.p,{children:["The ",(0,t.jsx)(i.strong,{children:"Human Protein Atlas"})," is indicated as a future enhancement, to bring in protein-level validation of expression findings."]}),(0,t.jsx)(i.p,{children:"By combining multiple curated biomedical datasets into a unified semantic framework, this project explores methods that support scalable data reuse, integration, and interpretability in research settings."})]}),"\n",(0,t.jsx)(i.h2,{id:"use-cases",children:"Use Cases"}),"\n",(0,t.jsx)(i.hr,{}),"\n",(0,t.jsxs)("div",{style:{textAlign:"justify"},children:[(0,t.jsx)(i.p,{children:"This integrated knowledge graph enables semantic exploration across compounds, kinase targets, gene identifiers, and expression profiles in normal and tumor tissues. Example use cases include:"}),(0,t.jsxs)(i.ul,{children:["\n",(0,t.jsx)(i.li,{children:"Prioritize kinase targets based on tumor-specific overexpression"}),"\n",(0,t.jsx)(i.li,{children:"Identify kinases with low expression in healthy tissues to reduce off-target toxicity"}),"\n",(0,t.jsx)(i.li,{children:"Rank compounds by target gene expression in selected cancer types"}),"\n",(0,t.jsx)(i.li,{children:"Filter targets by expression contrast across multiple tissues (e.g. GTEx vs. TCGA)"}),"\n",(0,t.jsx)(i.li,{children:"Explore compound-target coverage across tumor types for combination strategies"}),"\n"]})]})]})}function p(e={}){const{wrapper:i}={...(0,r.R)(),...e.components};return i?(0,t.jsx)(i,{...e,children:(0,t.jsx)(c,{...e})}):c(e)}}}]);